### PR TITLE
HIP for changing consensus group deselection dynamics

### PR DESCRIPTION
**Pull request by PaulVMo**
This HIP introduces a change to the consensus election dynamics on the Helium L1 blockchain to provide additional tunability for the deselection scoring of current consensus group members. The aim is to more quickly and reliable eject poor performers from the group.

Code for this change has already been merged as blockchain-core PR 1468 https://github.com/helium/blockchain-core/pull/1468

### CODE REVIEW

**Comment by vincenzospaghetti** [comment]
Hi @PaulVMo, does this need to be a HIP if it's already deployed? Please correct me if I have that wrong.

**Comment by abhay** [comment]
It's deployed but behind a chain variable that's bound to the 1.15.x release. I believe it's a fair thing to have a HIP and a quick vote on (at least by validator operators) since it changes election dynamics.

File: HIP-XXXX-consensus-deselection-history-weight.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+on the selection. This will help eject poor performers from the Consensus Group
+more quickly and maintain overall high performance of the chain.
+
+This chain var will be controllable at the discretion of the core developers as
+needed to promote performance of the blockchain. I propose starting with a
+value of 0 as the most effective way to measure success of this change. It is
+also suggested that the change be scheduled for early morning, i.e. 0800PST,
+early in the business week, to give the community and developers the ability to
+monitor actual impact and adjust the value up, if necessary.
+
```
Comment (vincenzospaghetti) [inline HIP-XXXX-consensus-deselection-history-weight.md:29]: Add here @abhay 's comment and a bit more context for why this is a HIP

Comment (vincenzospaghetti) [reply HIP-XXXX-consensus-deselection-history-weight.md:29]: Also add this change primarily affects Validator Operators/Owners

**Comment by vincenzospaghetti** [comment]
Okay. I likely see others having the same question I just asked. Can we put that in the context of this HIP? I added this as two comments to be added to the Summary portion. Otherwise looks great and, we'll get it out!

**Comment by PaulVMo** [comment]
@vincenzospaghetti great questions and feedback. I updated the summary to address the points you brought up. Please let me know if you have any more questions.

**Comment by abhay** [comment]
I’d be supportive of this coming to a full discussion (for a few days), some education from the author and the foundation, and a relatively quick vote. Ideally we could activate this variable around Nov 1.

**Comment by vincenzospaghetti** [comment]
All of that sounds good @abhay. I see this definitely needing some education from you and @PaulVMo. How can the Foundation help education? If you have a proposal of that we can add in the comments here that would be ideal.

Let's go ahead and open a channel.

**Pull request merged**